com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Prepass.cs

```diff
@@ -52,6 +52,12 @@ bool NeedClearGBuffer()
 
         HDUtils.PackedMipChainInfo GetDepthBufferMipChainInfo()
         {
+            // Initialize the MipChainInfo if nececcary. This will be necessary when rendering the first frame.
```

Comment:
Typo nececcary

Reply:
Thanks! it's funny how I had it correctly the second time I used the word in the same comment.